README.md

```diff
@@ -107,6 +106,103 @@ print(myShoppingList)
 // prints "["Almond Butter", "Jelly", "Bread", "Strawberries", "Bananas"]"
 ```
 
-* Challenge the student to write a function that takes in an `Array` as an argument, the `Array`'s type is `[String]`, where they are asked to grab the first element in the array (at index 0) and see if it's value is equal to "Bread". If so, return `true`.. if not, return `false`.
+- Challenge the student to write a function that takes in an `Array` as an argument, the `Array`'s type is `[String]`, where they are asked to grab the first element in the array (at index 0) and see if it's value is equal to "Bread". If so, return `true`.. if not, return `false`.
+
+--------------------------------------------------------------------------------
+
+## Arrays
+
+So far you have learnt about several types of variable to represent numbers, strings, and other values. This lesson will introduce a new type of variable, one that allows it to contain a _collection_ of values of the same type. These are called _arrays_.
+
```

Comment:
Before introducing arrays like this, I think we should set it up where we get the student THINKING about why an array exists and what problem it solves for them. 

For example, creating a narrative like this:

_So far we've used variables to store String and Int types. For example, we could create a variable like so:_
```swift_
let ingredient = "Jalapeno"
```
_However, variables only allow us to store one pice of information at a time. What if we were going shopping and needed to have all of the names of the ingredients that we wanted to make a sandwich? We technically could create multiple variables to represent this_

```swift
let ingredient1 =  "Bread"
let ingredient2 = "Butter"
let ingredient3 = "Cheese"
let ingredient4 = "Lettuce"
let ingredient5 = "Tomatoes" 
```

_But what's wrong with this approach? We could write a program that passes around these variables *one at a time*. This seems messy though. What if we forget about an ingredient or need to add a new ingredient. In the real world, we would write down all of our ingredients on one shopping list. Well, in Swift, we can do the same thing using an array._

In the example, we started with a "real world scenario" using a shopping list. Starting with something the student knows at this point (which was using variables of type String) and having them try to think about the solution using what it is they currently know (which was using variables). Then after they have the problem in their mind, they're able to see WHY arrays are useful and how they can use them in their own code. We are explicit in the struggle here, listing the ingredient variables so they can see it as opposed to just writing it down in a paragraph. 

This sort of primes them for all this new information which is about to be presented to them. I think they're most vulnerable at the very very beginning of the reading and if we can put them on a path knowing why they are to use something, they're able to understand the rest of the reading way more!